src/container/rpc.c

```diff
@@ -29,55 +29,25 @@
 #include "rpc.h"
 
 static int
-crt_proc_struct_rsvc_hint(crt_proc_t proc, struct rsvc_hint *hint)
+crt_proc_daos_epoch_range_t(crt_proc_t proc, daos_epoch_range_t *p)
 {
 	int rc;
 
-	rc = crt_proc_uint32_t(proc, &hint->sh_flags);
+	rc = crt_proc_memcpy(proc, p, sizeof(*p));
```

Comment:
Is that really faster than doing direct assignment of a 32-bit and 64-bit integers?

Reply:
I was much optimistic about this. It looks only copy whole structure starting with 4 members become faster with memcpy rather direct assignment each member.

So, I will restore for small structs.